--- tdesktop/Telegram/SourceFiles/ayu/sync/ayu_sync_controller.h 2023-07-09 13:59:32.938401231 +0300
+++ tdesktop/Telegram/SourceFiles/ayu/sync/ayu_sync_controller.h 2023-07-09 14:47:15.108199108 +0300
@@ -6,6 +6,7 @@
 // Copyright @Radolyn, 2023

 #pragma once
+#if 0

 #include "ayu/libs/json.hpp"
 #include "utils/ayu_pipe_wrapper.h"
@@ -38,3 +39,4 @@
     bool isAgentDownloaded();
     bool isAgentRunning();
 }
+#endif

--- tdesktop/Telegram/SourceFiles/ayu/sync/ayu_sync_controller.cpp 2023-07-20 21:38:52.919351141 +0300
+++ tdesktop/Telegram/SourceFiles/ayu/sync/ayu_sync_controller.cpp 2023-07-20 21:57:15.477084255 +0300
@@ -4,6 +4,7 @@
 // but be respectful and credit the original author.
 //
 // Copyright @Radolyn, 2023
+#if 0

 #include "ayu_sync_controller.h"
 #include "ayu/libs/process.hpp"
@@ -219,3 +220,4 @@
 		});
 	}
 }
+#endif

--- tdesktop/Telegram/SourceFiles/ayu/ui/settings/settings_ayu.cpp 2023-09-23 16:17:13.292320823 +0300
+++ tdesktop/Telegram/SourceFiles/ayu/ui/settings/settings_ayu.cpp 2023-09-23 16:15:57.247733049 +0300
@@ -912,6 +912,7 @@
 		});
 }

+#if 0
 void Ayu::SetupAyuSync(not_null<Ui::VerticalLayout *> container)
 {
 	AddSubsectionTitle(container, tr::ayu_AyuSyncHeader());
@@ -928,6 +929,7 @@
 						   controller->initializeAgent();
 					   });
 }
+#endif

 void Ayu::SetupSendConfirmations(not_null<Ui::VerticalLayout *> container)
 {
@@ -1017,6 +1019,7 @@
 	AddSkip(container);
 	AddDividerText(container, tr::ayu_SettingsCustomizationHint());

+#if 0
 	// todo: compilation flag
 	if constexpr (false) {
 		AddSkip(container);
@@ -1025,6 +1028,7 @@

 		AddDivider(container);
 	}
+#endif

 	AddSkip(container);
 	SetupSendConfirmations(container);

--- tdesktop/Telegram/SourceFiles/ayu/ui/settings/settings_ayu.h 2023-09-23 03:38:56.985513264 +0300
+++ tdesktop/Telegram/SourceFiles/ayu/ui/settings/settings_ayu.h 2023-09-23 03:40:48.800968044 +0300
@@ -57,7 +57,9 @@

 	void SetupRecentStickersLimitSlider(not_null<Ui::VerticalLayout *> container);

+#if 0
 	void SetupAyuSync(not_null<Ui::VerticalLayout *> container);
+#endif

 	void SetupSendConfirmations(not_null<Ui::VerticalLayout *> container);

--- tdesktop/Telegram/SourceFiles/ayu/sync/utils/ayu_pipe_wrapper.h 2023-09-23 03:41:48.122091690 +0300
+++ tdesktop/Telegram/SourceFiles/ayu/sync/utils/ayu_pipe_wrapper.h 2023-09-23 03:44:13.951516208 +0300
@@ -5,6 +5,7 @@
 //
 // Copyright @Radolyn, 2023
 #pragma once
+#if 0

 #include "ayu/libs/json.hpp"
 #include "ayu/libs/pipe.hpp"
@@ -26,3 +27,4 @@
 	std::unique_ptr<pipein> is;
 	std::unique_ptr<pipeout> os;
 };
+#endif

--- tdesktop/Telegram/SourceFiles/ayu/sync/utils/ayu_pipe_wrapper.cpp 2023-09-23 03:41:48.122091690 +0300
+++ tdesktop/Telegram/SourceFiles/ayu/sync/utils/ayu_pipe_wrapper.cpp 2023-09-23 03:43:12.847027992 +0300
@@ -4,6 +4,7 @@
 // but be respectful and credit the original author.
 //
 // Copyright @Radolyn, 2023
+#if 0

 #include "ayu_pipe_wrapper.h"
 #include <sstream>
@@ -63,3 +64,4 @@
 	auto p = json::parse(sb.str());
 	return p;
 }
+#endif

--- tdesktop/Telegram/SourceFiles/data/data_histories.cpp 2023-09-23 03:45:47.163276273 +0300
+++ tdesktop/Telegram/SourceFiles/data/data_histories.cpp 2023-09-23 03:50:07.087894795 +0300
@@ -202,7 +202,9 @@

 	Core::App().notifications().clearIncomingFromHistory(history);

+#if 0
 	AyuSync::getInstance().syncRead(history, tillId);
+#endif

 	// AyuGram sendReadMessages
 	const auto settings = &AyuSettings::getInstance();
