--- tdesktop/cmake/external/glibmm/CMakeLists.txt 2023-08-10 21:57:26.728120228 +0300
+++ tdesktop/cmake/external/glibmm/CMakeLists.txt 2023-08-10 22:02:28.563832199 +0300
@@ -8,8 +8,8 @@
 add_library(desktop-app::external_glibmm ALIAS external_glibmm)

 find_package(PkgConfig REQUIRED)
-pkg_check_modules(GLIBMM REQUIRED IMPORTED_TARGET glibmm-2.68>=2.77)
-pkg_check_modules(GIOMM REQUIRED IMPORTED_TARGET giomm-2.68>=2.77)
+pkg_check_modules(GLIBMM REQUIRED IMPORTED_TARGET glibmm-2.68>=2.76)
+pkg_check_modules(GIOMM REQUIRED IMPORTED_TARGET giomm-2.68>=2.76)

 if (DESKTOP_APP_USE_PACKAGED)
     target_link_libraries(external_glibmm
